README.md

```diff
@@ -34,17 +34,27 @@ Confidential emojivoto is build as a confidential computing application:
 
     Deploy with [MarbleRun CLI](https://www.marblerun.sh/docs/getting-started/quickstart/#step-1-install-the-cli)
 
-    * If you're running minikube on a machine that support SGX1+FLC (e.g.Azure Standard_DC*s)
+    * If you're running minikube on an Azure VM that supports SGX1+FLC (e.g.Azure Standard_DC*s)
 
-    ```bash
-    marblerun install
-    ```
+        ```bash
+        marblerun install
+        ```
+
+    * If you're running minikube on a generic SGX capable machine that supports SGX1+FLC
+
+        Assuming you have a PCCS reachable at `https://localhost:8081/sgx/certification/v3/`, install MarbleRun using the following command:
+
+        ```bash
+        marblerun install --dcap-qpl=intel --dcap-pccs-url="https://host.minikube.internal:8081/sgx/certification/v3/" --dcap-secure-cert="FALSE"
```

Comment:
The `host.minikube.internal` is exposed automatically by minikube?

Reply:
Yes, its a feature called [host access](https://minikube.sigs.k8s.io/docs/handbook/host-access/)